frontend/awx/common/util/getDocsBaseUrl.ts

```diff
@@ -1,10 +1,106 @@
+  notifiers: 'userguide/notifications.html',
+  topology: 'userguide/topology_viewer.html',
+  workflows: 'userguide/workflows.html',
+  eeMigration: 'upgrade-migration-guide/upgrade_to_ees.html',
+  jobTemplateSurveys: 'userguide/job_templates.html#surveys',
+  index: 'userguide/index.html',
+  hosts: 'userguide/hosts.html',
+  inventories: 'userguide/inventories.html',
+  constructedInventories: 'userguide/inventories.html#constructed-inventories',
+  managePlaybooksSC: 'userguide/projects.html#manage-playbooks-using-source-control',
+  projects: 'userguide/projects.html',
+  templates: 'userguide/job_templates.html',
+  // workflowTemplates: 'userguide/automation_controller_user_guide/controller-workflow-job-templates',
+  workflowVisualizer: 'userguide/workflow_templates.html#ug-wf-editor',
+  workflowVisBuild: 'userguide/workflow_templates.html#converge-node',
+  jobs: 'userguide/jobs.html',
+  schedules: 'userguide/scheduling.html',
+};
+
+const downstreamPaths: DocPathDictionary = {
+  credentialTypes: 'automation_controller_user_guide/assembly-controller-custom-credentials',
+  credentials: 'automation_controller_user_guide/controller-credentials',
+  organizations: 'automation_controller_user_guide/assembly-controller-organizations',
+  teams: 'automation_controller_user_guide/assembly-controller-teams',
+  users: 'automation_controller_user_guide/assembly-controller-users',
+  activityStream: 'automation_controller_user_guide/assembly-controller-activity-stream',
+  applications: 'automation_controller_user_guide/assembly-controller-applications',
+  executionEnvironments:
+    'automation_controller_user_guide/assembly-controller-execution-environments',
+  managementJobs: 'automation_controller_user_guide/assembly-controller-management-jobs',
+  notifiers: 'automation_controller_user_guide/assembly-controller-notifications',
+  topology: 'automation_controller_user_guide/assembly-controller-topology-viewer',
+  workflows: 'automation_controller_user_guide/assembly-controller-workflows',
+  eeMigration: 'red_hat_ansible_automation_platform_upgrade_and_migration_guide/upgrading-to-ees',
+  jobTemplateSurveys:
+    'automation_controller_user_guide/controller-job-templates#controller-surveys-in-job-templates',
+  index: 'automation_controller_user_guide/index',
+  hosts: 'automation_controller_user_guide/controller-hosts',
+  inventories: 'automation_controller_user_guide/controller-inventories',
+  constructedInventories:
+    'automation_controller_user_guide/controller-inventories#ref-controller-constructed-inventories',
+  managePlaybooksSC:
+    'automation_controller_user_guide/controller-projects#ref-projects-manage-playbooks-with-source-control',
+  projects: 'automation_controller_user_guide/controller-projects',
+  templates: 'automation_controller_user_guide/controller-job-templates',
+  // workflowTemplates: 'automation_controller_user_guide/controller-workflow-job-templates',
```

Comment:
Do we need this?

Reply:
No it was just a remanent removed the comment